packages/core-web/src/media/metrics.ts

```diff
@@ -78,9 +84,14 @@ export function addMediaMetrics(
 
     const { destroy: destroyListeners } = addEventListeners(element, store);
 
-    const { metrics, destroy: destroyMetrics } = addMediaMetricsToStore(store, {
+    const { destroy: destroyMetrics } = addMetricsToStore(store, {
       disableProgressListener: opts.disableProgressListener,
+      interval: opts.interval,
     });
+    const { destroy: destroyLegacyMetrics, metrics: legacyMetrics } =
```

Comment:
Calling attention to this - this is the function that our customers use to add metrics to a video element, and it still includes both the legacy metrics and the new metrics side-by-side. cc @leszko

Reply:
Good, we need both. The legacy + new one. Thanks for flagging this part.